Adding mutations for the double-odd problem

Sebastian42-5 opened the pull request:
For assignment 3 of SF-1

AlemoExe commented:
I dont think it can return empty list if the input is invalid (length < 2). You might want to add a clear message or exception for such cases. The rest seems good

Sebastian42-5 commented:
I already put that in the description of the problem

Sebastian42-5 commented on list-max-min/io.json:
```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "input": [3, 4, 6, 5],
```
Thank you. I fixed the problem

emaj9 commented:
But if the elements need to sum up to less than 30... why return more than just the sum of the largest and smallest number? like why return [7, 7, 7, 7] when [7] is also sums to less than 30? @Sebastian42-5

Sebastian42-5 commented:
> But if the elements need to sum up to less than 30... why return more than just the sum of the largest and smallest number? like why return [7, 7, 7, 7] when [7] is also sums to less than 30? @Sebastian42-5

The goal of my problem itself is to make a new list with copies of the sum of the smallest and the largest number of the original list until these numbers get as close to 30 as possible without exceeding 30. Maybe I just wasn't clear with the problem description I could change it if it's necessary

Monkey-D-Fabio reviewed (commented):
mutation 1 has  an unexpected indent in line 6, 7 and 8 for it to run properly

Sebastian42-5 commented:
> mutation 1 has an unexpected indent in line 6, 7 and 8 for it to run properly. Also third mutation does not produce an error.

I changed it thank you! I also changed mutation 3

emaj9 commented:
Kinda dicey only submitting on the lower end of the mutations, but all the mutations are quite high quality so well done.
